ocaml/xapi-storage-script/main.ml

```diff
@@ -120,7 +118,19 @@ let pvs_version = "3.0"
 
 let supported_api_versions = [pvs_version; "5.0"]
 
-let api_max = List.fold_left ~f:String.max supported_api_versions ~init:""
+let string_min a b =
+  if String.compare a b <= 0 then
+    a
+  else
+    b
+
+let string_max a b =
```

Comment:
I see you are keeping `Base`, and there is a `Base.String.{min,max}` that can be used.